packages/modules/web-viewer-react/package.json

```diff
@@ -90,6 +91,7 @@
     "@itwin/presentation-common": "rc",
     "@itwin/presentation-components": "rc",
     "@itwin/presentation-frontend": "rc",
+    "@itwin/reality-data-client": "latest",
```

Comment:
Is this temporary until release? I don't think we should use latest in package.json. We need a specific minimum that is supported. Template.json in the CRA templates is probably ok to use latest

Reply:
Yea i was going to go back and update all the deps to an actual version once theyre stable and officially released